tensorflow_app/views/layers_import.py

```diff
@@ -0,0 +1,530 @@
+    layer_params['num_output'] = kernel_shape[3]
+
+    pad_h, pad_w = get_padding(conv2d_op, kernel_shape, strides)
+    layer_params['pad_h'] = pad_h
+    layer_params['pad_w'] = pad_w
+
+    initializers = get_initializer_type(layer_ops)
+    try:
+        layer_params['weight_filler'] = initializers['kernel']
+        layer_params['bias_filler'] = initializers['bias']
+    except:
+        pass
+
+    return jsonLayer('Convolution', layer_params, get_input_layers(layer_ops), [])
+
+
+def import_conv3d(layer_ops):
+
+    conv3d_op = next((x for x in layer_ops if x.type == 'Conv3D'), None)
+
+    layer_params = {}
+
+    # search for weights and kernel ops to extract kernel shape and number of output
+    kernel_shape = [int(i) for i in conv3d_op.inputs[1].shape]
+
+    layer_params['kernel_d'] = kernel_shape[0]
+    layer_params['kernel_h'] = kernel_shape[1]
+    layer_params['kernel_w'] = kernel_shape[2]
+    layer_params['num_output'] = kernel_shape[4]
+
+    strides = [int(i) for i in conv3d_op.get_attr('strides')]
+
+    layer_params['layer_type'] = '3D'
+    layer_params['stride_d'] = strides[1]
+    layer_params['stride_h'] = strides[2]
+    layer_params['stride_w'] = strides[3]
+
+    initializers = get_initializer_type(layer_ops)
+    try:
+        layer_params['weight_filler'] = initializers['kernel']
+        layer_params['bias_filler'] = initializers['bias']
+    except:
+        pass
+
+    pad_d, pad_h, pad_w = get_padding(conv3d_op, kernel_shape, strides)
+    layer_params['pad_d'] = pad_d
+    layer_params['pad_h'] = pad_h
+    layer_params['pad_w'] = pad_w
+
+    return jsonLayer('Convolution', layer_params, get_input_layers(layer_ops), [])
+
+
+def import_deconvolution(layer_ops):
+    deconv_op = next((x for x in layer_ops if x.type == 'Conv2DBackpropInput'), None)
+
+    layer_params = {}
+
+    # search for weights and kernel ops to extract kernel shape and number of output
+    kernel_shape = [int(i) for i in deconv_op.inputs[1].shape]
+
+    layer_params['kernel_h'] = kernel_shape[0]
+    layer_params['kernel_w'] = kernel_shape[1]
+    layer_params['num_output'] = kernel_shape[3]
+
+    strides = [int(i) for i in deconv_op.get_attr('strides')]
+
+    layer_params['layer_type'] = '2D'
+    layer_params['padding'] = deconv_op.get_attr('padding')
+
+    pad_h, pad_w = get_padding(deconv_op, kernel_shape, strides)
+    layer_params['pad_h'] = pad_h
+    layer_params['pad_w'] = pad_w
+
+    initializers = get_initializer_type(layer_ops)
+    try:
+        layer_params['weight_filler'] = initializers['kernel']
+        layer_params['bias_filler'] = initializers['bias']
+    except:
+        pass
+
+    return jsonLayer('Deconvolution', layer_params, get_input_layers(layer_ops), [])
+
+
+def import_depthwise_convolution(layer_ops):
+    depthwise_conv_op = next((x for x in layer_ops if x.type == 'DepthwiseConv2dNative'), None)
+
+    layer_params = {}
+
+    if '3D' in depthwise_conv_op.type:
+        return
```

Comment:
We should throw some sort of error in such case so that user knows what was the issue